src/Month.js

```diff
@@ -264,6 +264,8 @@ class MonthView extends React.Component {
     let overlay = (this.state && this.state.overlay) || {}
     let { components } = this.props
 
+    let PopupComponent = components.popup || Popup
```

Comment:
should be `const` ?

Reply:
copied how the two props above this one was being used, but i confirmed they do use const so ill change it.